xla/service/hlo_verifier.cc

```diff
@@ -434,6 +434,7 @@ static Status CheckCommonAllGatherInvariants(HloInstruction* hlo,
                                                ag->use_global_device_ids()));
   TF_RETURN_IF_ERROR(CheckReplicaGroups(ag, group_mode));
   TF_RET_CHECK(ag->all_gather_dimension() >= 0);
+  TF_RET_CHECK(ag->operand_count() >= 1);
```

Comment:
@jurahul, @burmako, @radhakrishnaba I think this is correct. Let me know if you want me to remove this.

Reply:
Also seems ok to me.